gym/envs/classic_control/acrobot.py

```diff
@@ -340,6 +341,8 @@ def render(self, mode="human"):
 
     def close(self):
         if self.screen is not None:
+            import pygame
```

Comment:
I think this shouldn't be necessary? if `self.screen is not None`, then pygame would already be imported through `render()`

Reply:
In terms of actually running the code, it's not necessary, but any static code checker will freak out if this is not present